File: trunk/quests/gukbottom/a_ghoul_scribe.pl
# Ink of the dark - enchanter epic lead in quest

sub EVENT_ITEM {
  if (plugin::check_handin(\%itemcount, 10626 => 1)) {
    quest::say("Crroooaak! Crrroooooaaakk!");
    quest::summonitem(10601);
    quest::exp(10000);
  }
  else {
    plugin::return_items(\%itemcount);
  }
}

# EOF zone: gukbottom ID: 66170 NPC: a_ghoul_scribe

File: gukbottom/a_ghoul_scribe.lua
-- Ink of the dark - enchanter epic lead in quest

function event_trade(e)
	local item_lib = require("items");

	if(item_lib.check_turn_in(e.trade, {item1 = 10626})) then
		e.self:Say("Crroooaak! Crrroooooaaakk!");
		e.other:SummonItem(10601);
		e.other:Ding();
		e.other:AddEXP(10000);
	end
	item_lib.return_items(e.self, e.other, e.trade)
end

-- EOF zone: gukbottom ID: 66170 NPC: a_ghoul_scribe